### ISSUE TITLE
check how stdev are calculated

### ISSUE THREAD

**Issue by Valdes-Tresanco-MS** [Valdes-Tresanco-MS/gmx_MMPBSA#105]

**Comment by marioernestovaldes** [comment]
the standard deviation of a difference is not calculated as the std of a "difference" array. Instead, for a function _f_ = aA - bB, the standard deviation is approximately:

<img src="https://latex.codecogs.com/png.image?\dpi{110}&space;\sigma_{f}&space;\approx&space;&space;\sqrt{a^{2}&space;\sigma&space;_{A}^{2}&space;&plus;&space;b^{2}&space;\sigma&space;_{B}^{2}}" title="\sigma_{f} \approx \sqrt{a^{2} \sigma _{A}^{2} + b^{2} \sigma _{B}^{2}}" />

**Comment by Valdes-Tresanco-MS** [comment]
Example:
``` python
f = np.array([1, 2, 3, 4, 5])
f1 = np.array([6, 7, 8, 9, 10])
dfs = f1-f
```
Result
``` python
# average
f.mean() = 3.0
f1.mean() = 8.0
dfs.mean() = 5.0
f1.mean()-f.mean() = 5.0

# stdev
f.std() = 1.414
f1.std() =1.414
dfs.std() = 0.0  # Wrong way
sqrt(f1.std() ** 2 + f.std() ** 2)) = 2.0  # right way
```

**Comment by Valdes-Tresanco-MS** [comment]
Fixed via 7c620115

**Comment by Valdes-Tresanco-MS** [comment]
Previously, the stdev calculations for the sum of the DH and TDS were fixed, however, the way to calculate the stdev for compound terms such as GGAS, GSOL, or TOTAL remained incorrect.

Fixed via 6bf5565
